esp/xslt/esdl2monitor.xslt

```diff
@@ -21,6 +21,7 @@
     <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
     <xsl:param name="responseType" select="''"/>
     <xsl:param name="requestType" select="''"/>
+    <xsl:param name="diffmode" select="'Monitor'"/>
```

Comment:
default behavior is Monitor if diffmode not set, but does the tool ever set diffmode to 'Monitor'?